src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -53,6 +53,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       }
 
       request.setAttribute("profileOwner", user.getName());
+      request.setAttribute("OwnerID", user.getId());
```

Comment:
Please make the attribute name follow lower camelcase (https://google.github.io/styleguide/javaguide.html#s5.3-camel-case).